packages/create-amplify/src/tsconfig_initializer.ts

```diff
@@ -18,8 +18,11 @@ export class TsConfigInitializer {
     private readonly existsSync = _existsSync,
     private readonly execa = _execa
   ) {}
-  private readonly executableName =
-    process.env.PACKAGE_MANAGER_EXECUTABLE || 'npx'; // TODO: replace `process.env.PACKAGE_MANAGER_EXECUTABLE` with `getPackageManagerName()` once the test infra is ready.
+  private readonly executableName = !process.env.PACKAGE_MANAGER_EXECUTABLE
+    ? 'npx'
+    : process.env.PACKAGE_MANAGER_EXECUTABLE === 'yarn-stable'
+    ? 'yarn'
+    : process.env.PACKAGE_MANAGER_EXECUTABLE; // TODO: replace `process.env.PACKAGE_MANAGER_EXECUTABLE` with `getPackageManagerName()` once the test infra is ready.
```

Comment:
similar comment, here, it might be worth starting to package these things behind some abstraction. Keep env var as a mechanism, but figure out how rest of create-amplify is gonna interact with this.

Reply:
Since my next step is to get rid of env var and use a `getPackageManager` function, it's not necessary to refactor the env var here. Right?

Reply:
Yes. On the branch, you decide when and how to take this suggestion (as well as the other one).